Sources/Orbit/Components/InputField.swift

```diff
@@ -14,11 +14,12 @@ public struct InputField: View {
 
     @Binding private var value: String
     @Binding private var messageHeight: CGFloat
-    @State private var isPressed: Bool = false
     @State private var isEditing: Bool = false
 
     let label: String
     let placeholder: String
+    let prefix: Icon.Content
+    let suffix: Icon.Content
```

Comment:
What is the logic in naming this one suffix, but calling it disclosure in other components? If it's "because Orbit calls it that", then I suggest making them unify it, because, this is just confusing.

Reply:
Yep, so far it is the inconsistency in Orbit docs :/ I would address it later. For now, it seemed better if it matches the relevant component naming (although the disclosure is yet another term which should be renamed) 

![image](https://user-images.githubusercontent.com/35844477/152527871-54ffeae3-9cb4-4bf8-903b-8603ca9593bc.png)

![image](https://user-images.githubusercontent.com/35844477/152527899-675a110e-55d3-4e22-a8ae-19ad4d598dcc.png)